src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -295,3 +301,35 @@ def test_analyze_parse_child_logs(self, multihost, backupsssdconf):
             assert re.findall(r"RID#[0-9]*] Received error code", stdout)
         else:
             pytest.fail(f"{user} sucessful to login")
+
+    def test_non_root_privileged(self, multihost, backupsssdconf, localusers):
+        """
+        :title: SSSD: `sssctl analyze` command shouldn't require 'root' privileged
+        :id: 51a69d4e-7ae4-11ed-95a5-845cf3eff344
+        :description: `sssctl analyze` command shouldn't require
+            'root' privileged when run with `--logdir`
+            pointing to otherwise accessible files.
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2142960
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2142794
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2142961
+        :steps:
+          1. Create directory
+          2. Copy logs to above directory
+          3. Change ownership to a local user
+          4. Try sssctl analyze command pointing to above
+            directory with root and loacluser
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
```

Comment:
"Should succeed" does not really describe the expected result. Something like "sssctl analyze outputs provided for user root and user5000 are the same".

Reply:
fixed